scripts/build.mjs

```diff
@@ -108,6 +103,20 @@ async function run(argv) {
     throw new Error(`'${command}' failed with \n${stderr}`);
   }
 
+  // cjs for reexporting from commons only modules.
+  // If we need to rely more on this we can think about setting up a separate commonjs => commonjs build for .cjs files to .cjs
+  // `--extensions-.cjs --out-file-extension .cjs`
+  await cjsCopy({ from: srcDir, to: outDir });
```

Comment:
I don't understand what's the function of this 😅

Reply:
it copies .cjs files from the src folder to the build folder. we use them to import cjs only modules such as `next/document`.